test/e2e/actions/helm_related.go

```diff
@@ -29,10 +32,20 @@ func HelmDefaultUpgradeResouces(data *model.TestDataProvider) {
 			WaitDeployment(data.Resources, strconv.Itoa(generation+1))
 			ExpectWithOffset(1, data.Resources.ProjectID).ShouldNot(BeEmpty())
 		})
-		updatedDeployment := mongocli.GetDeploymentsInfo(data.Resources.ProjectID, data.Resources.Deployments[0].Spec.GetDeploymentName())
+		aClient := atlas.NewClientOrFail()
```

Comment:
We creating new client all each time, we can try to reuse the same one. Not critical thought and can be left as is :)

Reply:
Fixed!